src/test/java/com/amazon/opendistroforelasticsearch/sql/intgtest/PrettyFormatResponseTest.java

```diff
@@ -90,10 +90,17 @@ public void wrongIndexType() {
             String query = String.format("SELECT * FROM %s/%s", TestsConstants.TEST_INDEX_ACCOUNT, type);
             Protocol protocol = CheckPrettyFormatContents.execute(query, "jdbc");
         } catch (IllegalArgumentException e) {
-            assertThat(e.getMessage(), is(String.format("Index type %s does not exist", type)));
+            assertThat(e.getMessage(), is(String.format("Index type [%s/%s] does not exist", TestsConstants.TEST_INDEX_ACCOUNT, type)));
         }
     }
 
+    @Test
+    public void indexPattern() {
+        String query = "SELECT * FROM " + TestsConstants.TEST_INDEX + "_loc*";
+        Protocol protocol = CheckPrettyFormatContents.execute(query, "jdbc");
+        assertThat(4L, equalTo(protocol.getResultSet().getDataRows().getSize()));
```

Comment:
NP: I believe for ```assertThat()``` the format goes ```assertThat(T actual, Matcher<> matcher)``` so this assertion would be ```assertThat(protocol.getResultSet().getDataRows().getSize(), equalTo(4L))```. This way it almost reads like a sentence, "Assert that dataRows size is equal to 4".

Reply:
> NP: I believe for `assertThat()` the format goes `assertThat(T actual, Matcher<> matcher)` so this assertion would be `assertThat(protocol.getResultSet().getDataRows().getSize(), equalTo(4L))`. This way it almost reads like a sentence, "Assert that dataRows size is equal to 4".

Thanks! Fixing it. I think I confused the assertion of Hamcrest and JUnit. :)